lib/test_track_rails_client/rspec_helpers.rb

```diff
@@ -2,4 +2,8 @@
 
 RSpec.configure do |config|
   config.include TestTrackRailsClient::AssignmentHelper
+
+  config.before(:each) do
+    TestTrack::FakeServer.reset!(RSpec.configuration.seed)
```

Comment:
hmm. yea, i think i like stubbing that registry there. having this here in order to clean up after that stub method is very unclear to me.